### PR TITLE
Remove dependency on @mui/styles

### LINKED ISSUE #232: Need to remove @mui/styles peerDependency
#### Describe the bug
The package still has dependencies/peerDependencies on the old JSS @mui/styles package.

#### What is the expected behavior?
Our packages should be using all new emotion styles and not have a need for @mui/styles dependency anymore.

#### What are the steps to reproduce?
1. Look in the package.json
2. Look in the template file used to generate icons

#### Suggested fix
- Update the template to refactor out any need for mui/styles
- Remove any references to mui/styles including in package.json

#### Anything else to add?
This causes peerDependeny errors when installing the package via NPM because the @mui/styles package has conflicting dependency requirements for react 17.
This was reported by the Edge team.

### PR DESCRIPTION
**Pull request by daileytj**
Fixes #232. 

#### Changes proposed in this Pull Request:
- Remove dependency on @mui/styles

#### To Test:
- cd mui && yarn start
- Make sure the demo still works and all of the icons are rendered

### CODE REVIEW

**Comment by daileytj** [comment]
converting to a draft to refactor this in hopes of avoiding the addition of a peer-dependency (tss-react), which would result in needing a major upgrade

**Review by jeffvg — APPROVED** [review]
everything looks good for the icons, verified the 2 name changes

**Comment by emclaug2** [comment]
Whenever I run the commands `cd mui && yarn && yarn start` I see this type-related warning.  
image

Have you seen this before?  Anything missing from the instructions?

**Comment by daileytj** [comment]
@emclaug2 things are running fine on my end from a fresh clone

File: mui/CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,13 @@
 # Changelog
 
+## v4.0.0 (unreleased)
+
+### Updated
+
+-   Removed dependency on @mui/styles. [#232](https://github.com/brightlayer-ui/icons/issues/232)
+-   `SvgEvPlugIec62196T3a` icon name changed to `SvgEvPlugIec62196T3A`
```
Comment (joebochill) [inline mui/CHANGELOG.md:8]: The two renames should go under a 'Changed' sections, along with a mention of the new minimum required version of mui/material (whenever the path to that utility function was made)

**Pull request merged**